certified-connectors/Teams-Spirit/apiDefinition.swagger.json

```diff
@@ -0,0 +1,897 @@
+                  "expirationDateTime": {
+                    "type": "string",
+                    "description": "expirationDateTime"
+                  },
+                  "status": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "status"
+                  },
+                  "groupName": {
+                    "type": "string",
+                    "description": "groupName"
+                  },
+                  "groupDescription": {
+                    "type": "string",
+                    "description": "groupDescription"
+                  },
+                  "reason": {
+                    "type": "string",
+                    "description": "reason"
+                  },
+                  "rejectReason": {
+                    "type": "string",
+                    "description": "rejectReason"
+                  },
+                  "access": {
+                    "type": "boolean",
+                    "description": "access"
+                  },
+                  "reactionTime": {
+                    "type": "string",
+                    "description": "reactionTime"
+                  },
+                  "type": {
+                    "type": "string",
+                    "description": "type"
+                  },
+                  "tid": {
+                    "type": "string",
+                    "description": "tid"
+                  },
+                  "templateId": {
+                    "type": "string",
+                    "description": "templateId"
+                  },
+                  "attributeSettings": {
+                    "type": "array",
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "attributeId": {
+                          "type": "string",
+                          "description": "attributeId"
+                        },
+                        "value": {
+                          "type": "string",
+                          "description": "value"
+                        },
+                        "type": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "type"
+                        }
+                      }
+                    },
+                    "description": "attributeSettings"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get all Approvals",
+        "description": "Get all Approvals",
+        "operationId": "GetApprovals",
+        "parameters": []
+      }
+    },
+    "/approvals/{Approval-ID}/approve": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "Success",
+            "schema": {}
+          }
+        },
+        "summary": "Approve",
+        "description": "Approve",
```

Comment:
@tillkaemmerer we highly recommend having a different description than the summary. please follow our requirements [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).
Please fix in all your swagger file.